power/Power.cc

```diff
@@ -1204,7 +1204,14 @@ Power::findActivity(const Pin *pin)
     PwrActivity &activity = activity_map_[pin];
     if (activity.origin() != PwrActivityOrigin::unknown)
       return activity;
+  } else if (user_activity_map_.hasKey(pin)) {
+    PwrActivity &activity = user_activity_map_[pin];
+    if (activity.origin() != PwrActivityOrigin::unknown)
+      return activity;
   }
+  const char *pname = network_->pathName(pin);
+  if (strstr(pname, "/IQ") == nullptr)
+    std::cout << "WARNING: No Activity found for: " << pname << "\n";
```

Comment:
Can you use the native OpenSTA warning system instead? Cherry wouldn't accept this upstream.

Reply:
Adding a TODO to remove this code altogether, this is internal debugging while in dev mode.